### PR TITLE
translations PR 20161101

### PR DESCRIPTION
**Pull request by enahum**
Ideally this PR should be review by community members for each language.

Note: In the event that you see an english translation instead of the desired language is because this PR was submitted before the translation was made on the Mattermost Translation Server and probably will be submitted for the next PR.

### CODE REVIEW

**Comment by chikei** [comment]
zh-TW LGTM

**Comment by meilon** [comment]
de-DE looks fine

**Comment by kaakaa** [comment]
ja looks good

**Comment by aeomin** [comment]
zh_CN ok

File: i18n/pt-BR.json [hunk 1]
```diff
@@ -1532,8 +1532,8 @@
     "translation": "Falha ao importar: {{.DisplayName}}\r\n"
   },
   {
-    "id": "api.slackimport.slack_add_channels.import_failed.debug",
-    "translation": "Falha ao importar: %s"
+    "id": "api.slackimport.slack_add_channels.import_failed.warn",
+    "translation": "Slack Importer: Failed to import channel: %s"
```
Comment (cpanato) [inline i18n/pt-BR.json]: need to translate to Portuguese this one

**Comment by ArchRoller** [comment]
ru looks good.

**Comment by jasonblais** [comment]
Just needs a rebase and we're ready to merge this after one dev review.

**Pull request merged**